pyproject.toml

```diff
@@ -185,32 +185,9 @@ markers = [
     "integration_snowflake",
     "integration_mariadb",
     "integration_bigquery",
+    "integration_dynamodb",
     "integration_saas",
     "integration_saas_override",
-    "integration_mailchimp",
-    "integration_zendesk",
-    "integration_sentry",
-    "integration_stripe",
-    "integration_hubspot",
-    "integration_datadog",
-    "integration_domo",
-    "integration_doordash",
-    "integration_friendbuy_nextgen",
-    "integration_fullstory",
-    "integration_segment",
-    "integration_sendgrid",
-    "integration_friendbuy",
-    "integration_braintree",
-    "integration_auth0",
-    "integration_shopify",
-    "integration_square",
-    "integration_outreach",
-    "integration_recharge",
-    "integration_salesforce",
-    "integration_twilio_conversations",
-    "integration_adobe_campaign",
-    "integration_firebase_auth",
```

Comment:
Removing these old pytest markers for individual tests. There wasn't a need to continue doing this so I'm just cleaning them up.